From ebab50d92142daf70e3277b748bfb53964e79c56 Mon Sep 17 00:00:00 2001
From: Anastas Dancha <anapsix@random.io>
Date: Tue, 27 May 2014 17:11:42 +0000
Subject: [PATCH] [PATCH] add sqlite3 support Signed-off-by: Anastas Dancha
 <adancha@random.io>

---
 Gemfile      |    1 +
 Gemfile.lock |    2 ++
 2 files changed, 3 insertions(+)

diff --git a/Gemfile b/Gemfile
index 6bca027..757385e 100644
--- a/Gemfile
+++ b/Gemfile
@@ -18,6 +18,7 @@ gem 'activerecord-session_store'
 # DB
 gem 'mysql2', group: :mysql
 gem 'pg',     group: :postgres
+gem 'sqlite3',group: :sqlite3
 
 # Settings
 gem 'settingslogic'
diff --git a/Gemfile.lock b/Gemfile.lock
index 29d1690..25191b3 100644
--- a/Gemfile.lock
+++ b/Gemfile.lock
@@ -268,6 +268,7 @@ GEM
       actionpack (>= 3.0)
       activesupport (>= 3.0)
       sprockets (~> 2.8)
+    sqlite3 (1.3.9)
     stamp (0.5.0)
     state_machine (1.2.0)
     temple (0.6.7)
@@ -357,6 +358,7 @@ DEPENDENCIES
   simplecov
   sinatra
   slim
+  sqlite3
   stamp
   state_machine
   therubyracer
-- 
1.7.10.4

